### PR TITLE
Points system updates

### PR DESCRIPTION
**Pull request by ez314**
This update makes it possible to set up points system for future semesters without doing a complete find and replace (moves firebaseroot to config)

Also, update discord.js version required changes in a union type name.  

Sorry for changes in botconfig.example.js, i will no longer use vscode because i cant figure out the formatter :(

### CODE REVIEW

**Comment by ez314** [comment]
@ReshmiCode can you add @NickBurnett to this repo as well, want to get a CR from him for this and later, the circle feature

File: botconfig.example.js [hunk 1]
```diff
@@ -1,111 +1,113 @@
-const ENV = 'DEV';
```
Comment (NickBurnett) [inline botconfig.example.js:1]: real format

File: src/util/response.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { ColorResolvable, MessageEmbed, TextBasedChannels } from "discord.js";
+import { ColorResolvable, MessageEmbed, TextBasedChannel } from "discord.js";
```
Comment (NickBurnett) [inline src/util/response.ts:1]: Thank you `discord.js` for this wonderful renaming

**Review by NickBurnett — APPROVED** [review]
[👍🏼 I approve this 😎](https://www.google.com/url?sa=i&url=https%3A%2F%2Fgiphy.com%2Fexplore%2Fthumbs-ups&psig=AOvVaw3fYxN5agGMy4jpOS48gvJG&ust=1644895135456000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCIjc-Yae_vUCFQAAAAAdAAAAABAD)

**Pull request merged**